### ISSUE TITLE
 Error: No named parameter with name 'window'.

### ISSUE THREAD

**Issue by ged-flod** [SachinGanesh/screenshot#140]
After adding screenshot: `^1.3.0 to my pubspec.yaml`, when running the app on an Android device, I get the following error:

```
../.pub-cache/hosted/pub.dev/screenshot-1.3.0/lib/screenshot.dart:160:7: Error: No named parameter with name 'window'.
      window: ui.window,
      ^^^^^^
../../flutter/lib/src/rendering/view.dart:68:3: Context: Found this candidate, but arguments do not match.
  RenderView({
  ^^^^^^^^^^
target_kernel_snapshot failed: exception

FAILED: The build failed with an exception.
```
I'm using 

| Using | Release |
|-          |-            |
| Flutter | 3.10.0 |
| screenshot | 1.3.0 |
| OS | Ubuntu 22.04 |

**Comment by marcellocamara** [comment]
Unfortunately the owner is taking too long to fix it
Setting this at your `pubspec.yaml` will work:

```
  screenshot:
    git:
      url: https://github.com/SachinGanesh/screenshot.git
      ref: master
```

**Comment by h3x4d3c1m4l** [comment]
Fix seems to be published: https://pub.dev/packages/screenshot

**Comment by ritheshSalyan** [comment]
Published version `2.0.0` with support for flutter `3.10`

**Comment by ged-flod** [comment]
> Fix seems to be published: https://pub.dev/packages/screenshot

It works thanks.
